src/NRediSearch/Aggregation/AggregationBuilder.cs

```diff
@@ -0,0 +1,157 @@
+﻿// .NET port of https://github.com/RedisLabs/JRediSearch/
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using NRediSearch.Aggregation.Reducers;
+
+namespace NRediSearch.Aggregation
+{
+    public class AggregationBuilder
+    {
+        private readonly List<object> _args = new List<object>();
+
+        public bool IsWithCursor { get; private set; }
+
+        public ReadOnlyCollection<object> Args => new ReadOnlyCollection<object>(_args);
```

Comment:
ditto re "not free"; what is the use-case of this API? I wonder if it can be simply omitted, or exposed in a different way

Reply:
Yeah, the `ArgsString` was for testing, I believe I ported this because it was there. 

I'll go ahead and remove it. If it becomes useful to someone later I suppose we could just expose a property that returns the private `_args` List.